rusk-wallet/src/bin/io/prompt.rs

```diff
@@ -56,80 +75,44 @@ pub(crate) fn create_password(
-                // enter password
-                let q = Question::password("password")
-                    .message("Enter the password for the wallet:")
-                    .mask('*')
-                    .build();
-                let a = requestty::prompt_one(q)?;
-                let pwd1 = a.as_string().expect("answer to be a string");
-
-                // confirm password
-                let q = Question::password("password")
-                    .message("Please confirm the password:")
-                    .mask('*')
-                    .build();
-                let a = requestty::prompt_one(q)?;
-                let pwd2 = a.as_string().expect("answer to be a string");
-
-                // check match
-                pwds_match = pwd1 == pwd2;
-                if pwds_match {
-                    pwd = pwd1.to_string()
-                } else {
-                    println!("Passwords don't match, please try again.");
-                }
-            }
-            pwd
-        }
+        None => ask_pwd_confirmation()?,
     };
 
     Ok(hash(file_version, &pwd))
 }
 
 /// Display the recovery phrase to the user and ask for confirmation
+/// Recursively calls itself until the user confirms they have backed up
 pub(crate) fn confirm_recovery_phrase<S>(phrase: &S) -> anyhow::Result<()>
 where
     S: std::fmt::Display,
 {
     // inform the user about the mnemonic phrase
-    println!("The following phrase is essential for you to regain access to your wallet\nin case you lose access to this computer.");
-    println!("Please print it or write it down and store it somewhere safe:");
-    println!();
-    println!("> {}", phrase);
-    println!();
+    let msg = format!("The following phrase is essential for you to regain access to your wallet\nin case you lose access to this computer. Please print it or write it down and store it somewhere safe.\n> {} \nHave you backed up this phrase?", phrase);
 
     // let the user confirm they have backed up their phrase
-    loop {
-        let q = requestty::Question::confirm("proceed")
-            .message("Have you backed up your recovery phrase?")
-            .build();
-
-        let a = requestty::prompt_one(q)?;
-        if a.as_bool().expect("answer to be a bool") {
-            return Ok(());
-        }
+    let confirm = Confirm::new(&msg)
+        .with_help_message(
+            "It is important you backup the passphrase before proceeding",
+        )
+        .prompt()?;
+
+    if !confirm {
+        confirm_recovery_phrase(phrase)?
     }
+
+    Ok(())
 }
 
 /// Request the user to input the recovery phrase
 pub(crate) fn request_recovery_phrase() -> anyhow::Result<String> {
     // let the user input the recovery phrase
     let mut attempt = 1;
     loop {
-        let q = Question::input("phrase")
-            .message("Please enter the recovery phrase:")
-            .build();
+        let phrase = Text::new("Please enter the recovery phrase in English")
```

Comment:
wouldn't add the "in English". The recovery phrase is the recovery phrase and should be copied as is. If we decide to support other languages in the future, we don't want the user to translate the words.